storybookjs/storybook
# Docs: Implement Controls block

Telescoping on https://github.com/storybookjs/storybook/pull/20664

Issue: N/A

## What I did

- Very similar to #20664, implemented a controls block. There is some small duplication with the deprecated `ArgTable` component, but it is very simple.

- Use it in DocsPage

## How to test

- See stories
- Run an Autodocs page in a sandbox.


## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
